lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -761,10 +761,7 @@ def create_autoscaling_group(connection, module):
     target_group_arns = module.params['target_group_arns']
     availability_zones = module.params['availability_zones']
     launch_config_name = module.params.get('launch_config_name')
-    min_size = module.params['min_size']
```

Comment:
I think these need to be put back so the variables are always defined before creating an autoscaling group https://github.com/ansible/ansible/pull/27870/files#diff-da91568af8631cf0905f8dd43410a4d3L811